services/src/ogc/wfs/request.rs

```diff
@@ -0,0 +1,193 @@
+use crate::ogc::util::{parse_bbox, parse_time};
+use crate::util::from_str_option;
+use geoengine_datatypes::primitives::{BoundingBox2D, TimeInterval};
+use geoengine_datatypes::projection::Projection;
+use serde::{Deserialize, Serialize};
+
+// TODO: ignore case for field names
```

Comment:
Is there a way to achieve this with `serde` without implementing everything ourselves?

Reply:
there is the serde_aux library https://docs.rs/serde-aux/0.5.1/serde_aux/container_attributes/fn.deserialize_struct_case_insensitive.html

but this seems to require tagging each attribute. Also not sure how to apply this on the `#[serde(tag = "request")]`

Reply:
Mmh… difficult to achieve. Is it necessary to wrap it within another Struct?

Reply:
it doesn't seem to work with enums at all https://github.com/vityafx/serde-aux/issues/2

Reply:
I am planning some extension soon, if you are tracking this issue there, you'll probably see a notification once I am done with it. But now, yes, serde-aux doesn't support enums (there were some limitations about it when I was developing it a few years ago) and it required also to put it into a parent struct. I am sorry it can't be of help right now, but it should be soon.

Reply:
@vityafx thanks for letting us know that you are working on it. I'll keep track of the issue.